vumi_twilio_api/twilio_api.py

```diff
@@ -62,10 +79,28 @@ def format_xml(dct, root=None):
 
     @staticmethod
     def format_json(dct):
-        return json.dumps(dct)
+        if dct.get('Call'):
+            dct = dct['Call']
+        if dct.get('Version'):
+            dct = dct['Version']
```

Comment:
I couldn't find a Twilio example that has `Version` as a sub-tree of the XML so I'm just checking if this is needed.